scrape/scrape.go

```diff
@@ -273,6 +273,15 @@ func (sp *scrapePool) Sync(tgs []*targetgroup.Group) {
 		time.Since(start).Seconds(),
 	)
 	targetScrapePoolSyncsCounter.WithLabelValues(sp.config.JobName).Inc()
+
+	sp.mtx.RLock()
+	for _, t := range sp.targets {
+		tActive = append(tActive, t)
```

Comment:
tActive = append(tActive, sp.targets...)

Reply:
thanks

Reply:
this is a map so can't use the shorthand.